src/pages/contact_information.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  Card,
+  Image,
+  Text,
+  Button,
+  Group,
+  CardSection,
+} from "@mantine/core";
+
+interface ContactCardProps {
+  title: string;
+  operationTitle: string;
+  operationTime: string;
+  imgUrl: string;
+  email: string;
+}
+
+const imageUrl = "/assets/images/logos/registrar.png";
+
+const contactCards = [
+	{
+		imgUrl: imageUrl,
+		title: "Academic Support",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "ug_academic@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Registra",
```

Comment:
I think it's supposed to be Registrar? The email is `registrar.bkk@stamford.edu`

Reply:
oh yeaa, i didn't change it from tommy's version.